_scripts/instruction-widget/install.js

```diff
@@ -109,15 +109,12 @@ module.exports = function(context) {
   gentoo_install = function() {
     template = "gentoo";
 
-    context.package = "letsencrypt";
+    context.package = "certbot";
     context.base_command = "letsencrypt";
```

Comment:
To clarify: since the command that you'll be running from the command line will be certbot and not letsencrypt (presumably). So I'd change this line to say certbot as well, so that way the instructions that are generated will say something like
`$ certbot certonly -d www.example.com`
and so on, as opposed to
`$ letsencrypt certonly -d www.example.com`